be/src/vec/exec/format/csv/csv_reader.cpp

```diff
@@ -395,14 +328,10 @@ Status CsvReader::init_reader(bool is_load) {
     if (_enclose == 0) {
         text_line_reader_ctx = std::make_shared<PlainTextLineReaderCtx>(
```

Comment:
How about unify the naming of `PlainTextLineReaderCtx` and `EncloseCsvLineReaderContext`?
eg:
- PlainCsvLineReaderCtx
- EncloseCsvLineReaderCtx